src/util.c

```diff
@@ -645,8 +645,26 @@ printsocket(struct tcb *tcp, int fd, const char *path)
 	return true;
 }
 
+typedef bool (*scan_fdinfo_fn)(const char *value, void *data);
+
+static bool
+scan_fdinfo(pid_t pid_of_fd, int fd, const char *search_pfx,
+	    size_t search_pfx_len, scan_fdinfo_fn fn, void *data);
+
+#define is_ptmx(FINFO) ((FINFO)->type == FINFO_DEV_CHR \
+			&& (FINFO)->dev.major == 5 \
+			&& (FINFO)->dev.minor == 2)
+
+static bool get_pts(const char *value, void *data)
+{
+	struct finfo *finfo = data;
+
+	finfo->dev.tty_index = string_to_uint_ex(value, NULL, INT_MAX, "\n");
+	return true;
+}
```

Comment:
The longer I look at this function, the more it seems to me that `get_pts` is not the right name for it.  What do you think about `set_tty_index`?